.github/workflows/ci.yml

```diff
@@ -86,14 +80,7 @@ jobs:
         with:
           cache: true
       - name: Install dependencies
-        run: |
-          cd packages/dart
-          flutter pub get
-          cd ../..
-          cd packages/flutter
-          flutter pub remove parse_server_sdk
-          flutter pub add parse_server_sdk --path ../dart
-          flutter pub get
+        run: tools/flutter-dependencies.sh
```

Comment:
nice clean-up!

Reply:
Having the same part three times seemed hard to maintain.
I decided to not file a new PR, since nothing really changed.

Reply:
Sure, well done!